app/lib/pre_assembly/bundle.rb

```diff
@@ -27,11 +27,12 @@ class Bundle
              :path_in_bundle,
              :progress_log_file,
              :project_name,
-             :project_style,
+             :content_structure,
+             :object_discovery,
```

Comment:
I think this line was deliberately nuked?

Reply:
Confirmed: removed `object_discovery` and tests pass fine.  Delete it and put `:content_structure` in the correct alphabetical order, plz.

Reply:
yes please - alpha order.  "o" comes before "p" as well.